Dockerfile

```diff
@@ -29,6 +29,8 @@ FROM ghcr.io/talos-systems/xfsprogs:${PKGS} AS pkg-xfsprogs
 FROM ghcr.io/talos-systems/util-linux:${PKGS} AS pkg-util-linux
 FROM ghcr.io/talos-systems/kmod:${PKGS} AS pkg-kmod
 FROM ghcr.io/talos-systems/kernel:${PKGS} AS pkg-kernel
+FROM docker.io/andrewrynhard/u-boot:v0.3.0-32-g8d8eada AS pkg-u-boot
+# FROM ghcr.io/talos-systems/u-boot:${PKGS} AS pkg-u-boot
```

Comment:
do we need to merge the u-boot PR?

Reply:
We do, but that is failing because it fails for x86